src/modules/AddOnProduct.rb

```diff
@@ -405,6 +407,22 @@ def GetBaseProductURL
     #     "relurl://./Product_CD1/"
     #   ) -> "username:password@ftp://www.example.org/dir/Product_CD1/"
     def GetAbsoluteURL(base_url, url)
+      url_parsed = URI(url)
+
+      if url_parsed.scheme == "repo"
```

Comment:
The documentation comment needs updating.

Also, I'd like to see a *public* reference of the URIs supported, especially those with non-standard schemes like "relurl" and "repo".
Ideally a `yast-*/doc/*.md` file please :)

Reply:
I'm writing such a doc atm; any suggestion for its location welcome!